### PR TITLE
docs: Update Pull Request Template 

### PR DESCRIPTION
**Pull request by chinma-yyy**
## Proposed changes (including videos or screenshots)

image

image

## Steps to test or reproduce

Create a pull request to the Rocket Chat repository

### CODE REVIEW

**Comment by hugocostadev** [comment]
Thanks a lot for your contribution @chinma-yyy !  We made a mistake by forgetting to update the template 💯 🚀

**Review by ggazzo — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-  [IMPROVE] For an improvement (performance or little improvements) in existing features
-  [FIX] For bug fixes that affect the end-user
-  [BREAK] For pull requests including breaking changes
-  Chore: For small tasks
-  Doc: For documentation
+  feat: Adding a new feature
+  refactor: For an improvement (performance or little improvements) in existing features
+  fix: For bug fixes that affect the end-user
+  chore: For small tasks
+  doc: For documentation
```
Comment (sampaiodiego) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: the correct is `docs` .. not sure if we should change to `doc` though.. 🤔 
```suggestion
  docs: For documentation
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 
 <!-- Your Pull Request name should start with one of the following tags
-  [NEW] For new features
-  [IMPROVE] For an improvement (performance or little improvements) in existing features
-  [FIX] For bug fixes that affect the end-user
-  [BREAK] For pull requests including breaking changes
-  Chore: For small tasks
-  Doc: For documentation
+  feat: Adding a new feature
+  refactor: For an improvement (performance or little improvements) in existing features
```
Comment (sampaiodiego) [inline .github/PULL_REQUEST_TEMPLATE.md:5]:
```suggestion
  refactor: A code change that doesn't change behavior (it doesn't add anything and doesn't fix anything)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+  fix: For bug fixes that affect the end-user
+  chore: For small tasks
+  doc: For documentation
+  ci: For updating CI configuration
+  test: For adding tests
+  i18n: For updating any translations
+  regression: For fixing any regression
```
Comment (ggazzo) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
  regression: Issues created/reported/fixed during the development phase. kind of problem that never existed in production and that we don't need to list in a changelog for the end user
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ggazzo) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: Conventional commits mentions `docs` 🤷

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (chinma-yyy) [reply .github/PULL_REQUEST_TEMPLATE.md]: Ohh, actually the previous template had no explanation and I had no idea about the issue type, so I just added it. Thank you

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**